.github/workflows/pr-checks.yml

```diff
@@ -38,59 +26,11 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
-      - name: ktlint
-        run: ./gradlew :stream-chat-android-offline:ktlintCheck
-      - name: Unit tests
-        run: ./gradlew :stream-chat-android-offline:testDebug --stacktrace
-      - name: Coverage
-        run: ./gradlew :stream-chat-android-offline:jacocoTestReport
-      - name: Upload Coverage
-        uses: codecov/codecov-action@v1
-      - name: Upload testDebugUnitTest results
-        uses: actions/upload-artifact@v2
-        if: failure()
-        with:
-          name: testDebugUnitTest
-          path: stream-chat-android-offline/build/reports/tests/testDebugUnitTest
-
-  stream_chat_android:
-    name: stream-chat-android
-    runs-on: ubuntu-18.04
-    steps:
-      - name: Check out code
-        uses: actions/checkout@v2
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
-        with:
-          java-version: 1.8
-      - name: ktlint
-        run: ./gradlew :stream-chat-android:ktlintCheck :stream-chat-android-sample:ktlintCheck
-      - name: Unit tests
-        run: ./gradlew :stream-chat-android:testDebug :stream-chat-android-sample:testDebug --stacktrace
-      - name: Upload testDebugUnitTest results
-        uses: actions/upload-artifact@v2
-        if: failure()
-        with:
-          name: testDebugUnitTest
-          path: stream-chat-android/build/reports/tests/testDebugUnitTest
-
-  stream_chat_android_ui_components:
-    name: stream-chat-android-ui-components
-    runs-on: ubuntu-18.04
-    steps:
-      - name: Check out code
-        uses: actions/checkout@v2
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
-        with:
-          java-version: 1.8
-      - name: ktlint
-        run: ./gradlew :stream-chat-android-ui-components:ktlintCheck :stream-chat-android-ui-components-sample:ktlintCheck
       - name: Unit tests
-        run: ./gradlew :stream-chat-android-ui-components:testDebug :stream-chat-android-ui-components-sample:testDebug --stacktrace
+        run: ./gradlew testDebug --stacktrace
       - name: Upload testDebugUnitTest results
         uses: actions/upload-artifact@v2
         if: failure()
         with:
           name: testDebugUnitTest
-          path: stream-chat-android-ui-components/build/reports/tests/testDebugUnitTest
+          path: /**/build/reports/tests/testDebugUnitTest
```

Comment:
It will break tests report, right?
I mean, some files will be overridden

Reply:
Or it will keep folder structure?

Reply:
I think that should be okay, [the docs for the Action](https://github.com/actions/upload-artifact#upload-using-multiple-paths-and-exclusions) say this (see also the example below this in the docs):

> If a wildcard pattern is used, the path hierarchy will be preserved after the first wildcard pattern.

But I'll break some tests and verify that it works correctly.

Reply:
This is the structure in the zip produced, I think we're good: 

![image](https://user-images.githubusercontent.com/12054216/100991286-1ffa7f00-3553-11eb-8de8-a4b529bf3fa5.png)

Reply:
That artifact is here https://github.com/GetStream/stream-chat-android/pull/942/checks?check_run_id=1491606030